lib/app/index.js

```diff
@@ -66,11 +196,10 @@ exports.get = function (name) {
 /**
  * Returns all app instances in a list
```

Comment:
Small detail, but I would perhaps take the opportunity to say that you now return a copy of the list, not the list itself (just to clarify the use of `slice()` here)